perf/benchmark/runner/runner.py

```diff
@@ -363,36 +362,22 @@ def run(self, headers, conn, qps, size, duration):
                     frequency=self.frequency)
 
 
-PERFCMD = "/usr/lib/linux-tools/4.4.0-131-generic/perf"
-FLAMESH = "flame.sh"
-PERFSH = "get_perfdata.sh"
-PERFWD = "/etc/istio/proxy/"
-
 WD = os.getcwd()
 LOCAL_FLAMEDIR = os.path.join(WD, "../flame/")
-LOCAL_FLAMEPATH = LOCAL_FLAMEDIR + FLAMESH
-LOCAL_PERFPATH = LOCAL_FLAMEDIR + PERFSH
```

Comment:
Was this code added to support flamegraphs using bcc ?
I am ok fixing the mainline path right now, but we need to restore the other path if it is broken.

Reply:
Yes, actually they are the same thing, this fix is just to simplify the old process by calling the newly added `get_proxy_perf.sh`, the `get_proxy_perf.sh` has same logic as what we did previously.